pkg/sync/syncer.go

```diff
@@ -108,8 +108,15 @@ func shouldWaitAndRetry(ctx context.Context, err error) bool {
 		details := st.Details()
 		for _, detail := range details {
 			if rlData, ok := detail.(*v2.RateLimitDescription); ok {
-				wait = time.Until(rlData.ResetAt.AsTime())
-				wait /= time.Duration(rlData.Limit)
+				waitAtRest := time.Until(rlData.ResetAt.AsTime())
```

Comment:
Nitpick: I think this should be called waitResetAt or something, not Rest.

Reply:
that's actually a typo from my part lol